scripts/bfb-install

```diff
@@ -568,19 +566,14 @@ cleanup() {
     fi
   fi
 
-  if [ $runtime -eq 1 ]; then
+  if [ $deferred -eq 1 ]; then
     # Reset to default state.
     echo "BOOT_RESET_SKIP 0" > ${rshim_node}/misc
 
     # Cleanup SP2.BIT2.
     sp2=`${BF_REG} $(basename ${rshim_node}) ${RSH_SCRATCHPAD2}.64 | awk '{print $3}'`
     sp2=$((sp2 & ~4))
     ${BF_REG} $(basename ${rshim_node}) ${RSH_SCRATCHPAD2}.64 $sp2 >/dev/null
-
-    # Set SWINT2.BIT2 for DPU mode
-    if [ ${nic_mode} -eq 0 ]; then
-      ${BF_REG} $(basename ${rshim_node}) ${RSH_SWINT}.64 0x4 >/dev/null
-    fi
```

Comment:
I think we should update line 577 instead to match the bits in UEFI ACPI, and leave these several lines